common/models/song.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+const { getSong } = require('../../server/utils/song');
+const { getMoreMusicFromSpotify} = require('../../server/utils/getMoreFromSpotify');
+
+module.exports = function(Song) {
+  Song.getTrackData = function(songUri, userID, cb) {
+    getSong(songUri, userID)
+    .then(song => cb(null, song))
```

Comment:
Don't forget to catch this!
```suggestion
    .then(song => cb(null, song))
    .catch(err => cb(err));
```